### ISSUE TITLE
Adding PR/ISSUE Templates for this repo

### ISSUE THREAD

**Issue by 5h0ov** [SFTtech/openage#1558]
Greetings! 
I would like to add some PR and ISSUE templates for this repo if possible.

You can tell me what to categorize like Bug Reports, New Feature... like that.

In PR there would be the Issue Number(if applicable), Description, Screenshots(if any) and other stuff like checkboxes that state the contributor has read contributing guidelines.

If possible kindly assign me this issue under hacktoberfest! Thanks!

**Comment by heinezen** [comment]
Sure, this wouldn't hurt :)

Can you try using the new [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) for this? The old template format is probably also fine.

**Comment by TheJJ** [comment]
We didn't use issue templates so far since we had the time to answer to each request individually, so people don't need to be restricted by some random input form. If we find that some form is useful with github's current implementation restrictions, we should always allow free form input - I personally don't like filling forms which request infos i'm very certain not useful for the request.

As long as we can easily request followup infos in an interactive way, I would rather not use issue form fields 🥲

**Comment by heinezen** [comment]
@TheJJ Yeah maybe you're right on this. I wouldn't mind issue/PR templates though, especially since sometimes new contributors seem to not have a clue what to do/write, e.g. they forget to add their name to the copying file...

**Comment by TheJJ** [comment]
Yea, so if we can do it in a optional way that's attractive for inexperienced users, I think we could benefit.

**Comment by 5h0ov** [comment]
> Sure, this wouldn't hurt :)
> 
> ~~Can you try using the new [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) for this? The old template format is probably also fine.~~ (disregard that, see below)

hmmm ok will see what i can do!

**Comment by 5h0ov** [comment]
> Yea, so if we can do it in a optional way that's attractive for inexperienced users, I think we could benefit.

ye sure there be can the normal blank template alongwith it

**Comment by simonsan** [comment]
> We didn't use issue templates so far since we had the time to answer to each request individually, so people don't need to be restricted by some random input form. If we find that some form is useful with github's current implementation restrictions, we should always allow free form input - I personally don't like filling forms which request infos i'm very certain not useful for the request.
> 
> As long as we can easily request followup infos in an interactive way, I would rather not use issue form fields 🥲

Issue forms or templates don't limit this experience at all, they give a guideline rather. As you can see on the screenshot, you can still choose a blank issue without any constrains. So it's an addition and not a limitation to the current process. It just guides someone new through the process of reporting/articulating their problem enough, to make themselves understood to the devs of the software.

![2024-03-10 00_42_59-New Issue · pace-rs_pace — Mozilla Firefox](https://github.com/SFTtech/openage/assets/14062932/cb1f6de3-1eaf-445d-b8e2-1c6a4dc0185b)

**Comment by TheJJ** [comment]
I'll close this for now - if anyone proposes to add a nice template via pull request, we'll of course consider deploying it.
